src/hivpy/demographics.py

```diff
@@ -177,7 +184,7 @@ def __init__(self, **kwargs):
     def initialize_sex(self, count):
         sex_distribution = (
             self.params['female_ratio'], 1 - self.params['female_ratio'])
-        return np.random.choice(SexType.categories, count, sex_distribution)
+        return pd.Series(np.random.choice(SexType, count, sex_distribution)).astype(SexDType)
```

Comment:
So in the end was the performance hit from the categorical not significant? I lost track of where we left it!

Reply:
There was a bit of a performance hit, but it doesn't seem to be a big thing. `update_sex_groups` is also sex dependent but doesn't seem to be slowing things down very much, so I wonder if the extra time that it took was set-up time rather than performance time for using it when looking things up etc. 
I don't have strong feelings about using it either way though.